dot/parachain/types/types.go

```diff
@@ -740,6 +747,54 @@ type Subsystem interface {
 	Stop()
 }
 
+type Present struct{}
```

Comment:
Add a comment to explain this type. also Do we need this type in multiple subsystems?

Reply:
This is a general type that is used by `Constraints`